tests/core/arch-arm/stalker-arm.c

```diff
@@ -69,6 +69,8 @@ TESTLIST_BEGIN (stalker)
   TESTENTRY (thumb_tbb)
   TESTENTRY (thumb_tbh)
 
+  TESTENTRY (self_modifying)
```

Comment:
New tests should be named stating the expectation, e.g. something like `self_modifying_code_should_be_detected`.